modular_zubbers/code/modules/synths/code/robot_mending.dm

```diff
@@ -0,0 +1,300 @@
+	. = ..()
+	if(.)
+		var/obj/item/bodypart/targeted_bodypart = target.get_bodypart(user.zone_selected)
+		return(targeted_bodypart.get_wound_type(targetable_wound))
+
+//SURGERY STEPS
+
+///// Reset Detached Fastenings (Severe)
+/datum/surgery_step/reset_detached_fastenings
+	name = "reset internal components (crowbar)"
+	implements = list(
+		TOOL_CROWBAR = 100,
+		/obj/item/bonesetter = 50,
+		/obj/item/stack/sticky_tape/surgical = 30,
+		/obj/item/stack/sticky_tape/super = 20,
+		/obj/item/stack/sticky_tape = 10)
+	time = 20
+
+/datum/surgery_step/reset_detached_fastenings/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	if(surgery.operated_wound)
+		display_results(
+			user,
+			target,
+			span_notice("You begin to reset the internal components in [target]'s [parse_zone(user.zone_selected)]..."),
+			span_notice("[user] begins to reset the internal components in [target]'s [parse_zone(user.zone_selected)] with [tool]."),
+			span_notice("[user] begins to reset the internal components in [target]'s [parse_zone(user.zone_selected)]."),
+		)
+		display_pain(target, "The aching pain in your [parse_zone(user.zone_selected)] is overwhelming!")
+	else
+		user.visible_message(span_notice("[user] looks for [target]'s [parse_zone(user.zone_selected)]."), span_notice("You look for [target]'s [parse_zone(user.zone_selected)]..."))
+
+/datum/surgery_step/reset_detached_fastenings/success(mob/living/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery, default_display_results = FALSE)
+	if(surgery.operated_wound)
+		if(isstack(tool))
+			var/obj/item/stack/used_stack = tool
+			used_stack.use(1)
+		display_results(
+			user,
+			target,
+			span_notice("You successfully reset the internal components in [target]'s [parse_zone(target_zone)]."),
+			span_notice("[user] successfully resets the internal components in [target]'s [parse_zone(target_zone)] with [tool]!"),
+			span_notice("[user] successfully resets the internal components in [target]'s [parse_zone(target_zone)]!"),
+		)
+		log_combat(user, target, "reset detached fastenings in", addition="COMBAT MODE: [uppertext(user.combat_mode)]")
+	else
+		to_chat(user, span_warning("[target] has no detached fastenings there!"))
+	return ..()
+
+/datum/surgery_step/reset_detached_fastenings/failure(mob/user, mob/living/target, target_zone, obj/item/tool, datum/surgery/surgery, fail_prob = 0)
+	..()
+	if(isstack(tool))
+		var/obj/item/stack/used_stack = tool
+		used_stack.use(1)
+
+///// Repair Detached fastenings (Severe)
+/datum/surgery_step/repair_detached_fastenings
+	name = "repair detached_fastenings (wrench/welder)"
+	implements = list(
+		TOOL_WRENCH = 100,
+		TOOL_WELDER = 100,
+		/obj/item/bonesetter = 50,
+		/obj/item/stack/medical/bone_gel = 50,
+		/obj/item/stack/sticky_tape/surgical = 50,
+		/obj/item/stack/sticky_tape/super = 25,
+		/obj/item/stack/sticky_tape = 15,)
+	time = 20
```

Comment:
20 what? use SECONDS define please

Reply:
Original code I pulled this from did not have seconds define but will add

Reply:
20 means 2 seconds